src/components/routes/Booking/BookingOptions/SelectPaymentOption/SelectPaymentOption.tsx

```diff
@@ -45,6 +49,7 @@ class SelectPaymentOption extends React.Component<Props> {
                   Select Payment Type
                 </option>
                 {showBee && <option value={Currency.BEE}>BEE</option>}
+                {showBee && <option value={Currency.DAI}>DAI</option>}
```

Comment:
small comment, but how do we want to order the select box? from the options we want them to select the most to least?

Reply:
Good question! I'm not sure additional ERC-20 currencies belong in this select box at all, particularly as we grow to accept more. In lieu of trying to tackle that design problem I'm trying to fit it in with minimal deviation from the existing UI.

For now, I've ordered it alphabetically and welcome any other guidance or direction.